### PR TITLE
Shorten nightly build to be under 1h

### PR DESCRIPTION
**Pull request by xadupre**

### CODE REVIEW

File: .azure-pipelines/linux-CI-nightly.yml [hunk 1]
```diff
@@ -114,12 +114,12 @@ jobs:
       pytest tests --durations=0 --basetemp=temp --doctest-modules --junitxml=junit/test-results.xml
     displayName: 'pytest'
 
-  - script: |
-      pip install onnxmltools openpyxl
-      coverage run --include=skl2onnx/** tests/benchmark.py
-      coverage report -m
-      coverage html
-    displayName: 'coverage'
+#  - script: |
```
Comment (prabhat00155) [inline .azure-pipelines/linux-CI-nightly.yml]: Why don't we just remove it instead?

Comment (xadupre) [reply .azure-pipelines/linux-CI-nightly.yml]: To avoid searching for the command line if we need to put it back.

Comment (prabhat00155) [reply .azure-pipelines/linux-CI-nightly.yml]: Don't we have the same statements in the CI build files?

Comment (wenbingl) [reply .azure-pipelines/linux-CI-nightly.yml]: I prefer to clean them up as well. you can check the git history if you want to see it again in the future.

Comment (xadupre) [reply .azure-pipelines/linux-CI-nightly.yml]: I'll do it but I doubt anybody will think of looking at the history to get this command line.

**Pull request merged**